packages/holochain/versions/update_config.toml

```diff
@@ -0,0 +1,17 @@
+[main]
+git-rev = "branch:main"
+
+[develop]
+git-rev = "branch:develop"
+
+[develop_lair_0_1]
+git-rev = "branch:develop"
+lair-version-req = "~0.1"
+
+[develop_lair_0_0]
+git-rev = "branch:develop"
+lair-version-req = "~0.0"
+
+[v0_0_110]
+git-rev = "commit:holochain-0.0.110"
```

Comment:
How is holochain-0.0.110 a commit? Is that a git tag or branch name?

Reply:
it's a tag and `commit` is a misnomer that i renamed to `manual`. so this is  a leftover and actually a bug, thanks for catching :+1:

Reply:
I don't think I like `git-rev=manual:X`. It's not necessarily manual anyway since it could be automatically generated, and specifying a tag manually is no different from specifying a branch manually, so `branch` vs `manual` seems weird. Is it just for git tags? Do commit hashes work too? If they do, it sounds like the distinction is about whether or not it's frozen at a specific git revision or updating to the latest value of a git ref/branch. Maybe call it `branch` vs `revision` or `branch` vs `frozen` or `ref` vs `rev`

Reply:
i agree, i don't particularly like the name. however i adopted it from [nvfetcher](https://github.com/berberman/nvfetcher#nvchecker) as it translates to that directly.

i'm happy to receive suggestions and i'm also open to rename `--git-rev` to something more accurate and expressive.

Reply:
it's now this:

```
        --git-src <git-src>
            Git revisin specifier for fetching the holochain sources. Either: branch:<branch_name> or revision:<id>
```